tests/longevity-staging.yaml

```diff
@@ -61,6 +61,21 @@ backends: !mux
             ami_db_cassandra_user: 'ubuntu'
             ami_id_monitor: 'ami-29035f52'
             ami_monitor_user: 'centos'
+    docker: !mux
+        cluster_backend: 'docker'
+        docker_image: 'scylladb/scylla'
+        user_credentials_path: '~/.ssh/scylla-test'
+    baremetal: !mux
+        cluster_backend: 'baremetal'
+        user_credentials_path: '~/.ssh/scylla-test'
+        # all the below are lists of physical machine IPs
+        db_nodes_public_ip: []
+        db_nodes_private_ip: []
+        loaders_public_ip: []
+        loaders_private_ip: []
+        monitor_nodes_public_ip: []
+        monitor_nodes_private_ip: []
+        scylla_repo: 'https://s3.amazonaws.com/downloads.scylladb.com/rpm/unstable/centos/branch-2.0/38/scylla.repo'
```

Comment:
Can we replace the url with: `scylla_repo: 'REPO_FILE_PATH'`?

Reply:
o yes